src/libPMacc/include/algorithms/math/doubleMath/floatingPoint.tpp

```diff
@@ -60,6 +72,29 @@ struct Float2int_rd<double>
     }
 };
 
+template<>
+struct Float2int_rn<double>
+{
+    typedef int result;
+
+    HDINLINE result operator( )(double value)
+    {
+#if __CUDA_ARCH__
+        return ::__double2int_rn( value );
+#else
+        if(value < 0.0)
+            return -(*this)(-value);
+        static const result epsilon = 1e-7;
+        /* Round towards nearest with .5 -> +inf */
```

Comment:
This would make this comment more clear. Because currently it sounds like `static_cast` rounds to nearest, which would be wrong:
```
static_cast(-5.8) --> -5
static_stast(-5.2) --> -5
static_cast(+5.2) --> +5
static_cast(+5.8) --> +5
```